src/test/java/seedu/address/model/person/NameContainsKeywordsPredicateTest.java

```diff
@@ -70,6 +70,6 @@ public void test_nameDoesNotContainKeywords_returnsFalse() {
         // Keywords match deadline, email and module code, but does not match name
         predicate = new NameContainsKeywordsPredicate(Arrays.asList("12345", "alice@email.com", "Main", "Street"));
         assertFalse(predicate.test(new PersonBuilder().withName("Alice").withDeadline("20-02-2020 2002")
-                .withEmail("alice@email.com").withModuleCode("CS2103T").build()));
+                .withModuleCode("CS2103T").build()));
```

Comment:
This test case seems to still have references to email.

Reply:
edited!